### PR TITLE
Add template lag to troubleshooting

### PR DESCRIPTION
**Pull request by Piphi5**
The current FRC-Char template prints the number of data samples collected and the time spent collecting after exiting autonomous. This just adds some troubleshooting options that can be done utilizing this feature.

### CODE REVIEW

File: source/docs/software/wpilib-tools/robot-characterization/viewing-diagnostics.rst [hunk 1]
```diff
@@ -51,10 +51,10 @@ This plot is interactive, and may be rotated by clicking-and-dragging. The quasi
 
 The discontinuity corresponds to ``kS``, which always opposes the direction of motion and thus changes direction as the plot crosses the 0 velocity mark.
 
-Diagnostics Plots for Common Failure Modes
+Common Failure Modes
 ------------------------------------------
```
Comment (Daltz333) [inline source/docs/software/wpilib-tools/robot-characterization/viewing-diagnostics.rst:55]: Title overline here

**Review by Daltz333 — CHANGES_REQUESTED** [review]

**Comment by Daltz333** [comment]
Needs a rebase

**Comment by Piphi5** [comment]
> Needs a rebase

Done.

File: source/docs/software/wpilib-tools/robot-characterization/viewing-diagnostics.rst [hunk 2]
```diff
@@ -51,10 +51,10 @@ This plot is interactive, and may be rotated by clicking-and-dragging. The quasi
 
 The discontinuity corresponds to ``kS``, which always opposes the direction of motion and thus changes direction as the plot crosses the 0 velocity mark.
 
-Diagnostics Plots for Common Failure Modes
-------------------------------------------
+Common Failure Modes
+~~~~~~~~~~~~~~~~~~~~
```
Comment (Daltz333) [inline source/docs/software/wpilib-tools/robot-characterization/viewing-diagnostics.rst:55]:
```suggestion
^^^^^^^^^^^^^^^^^^^^
```

**Review by Daltz333 — CHANGES_REQUESTED** [review]

**Review by Daltz333 — APPROVED** [review]

File: source/docs/software/wpilib-tools/robot-characterization/viewing-diagnostics.rst [hunk 3]
```diff
@@ -99,3 +99,10 @@ Magnetic Encoders Velocity Noise
...
 
 Characterization constants can sometimes be accurately determined even from data polluted this noise by increasing the accel window size setting.  However, this sort of encoder noise is problematic for robot code much the same way it is problematic for the characterization tool.  As the root cause of the noise is not known, it is recommended to try a different encoder setup if this is observed, either by moving the encoders to a different shaft or replacing them with a different type of encoder.
+
+Template Lag
+^^^^^^^^^^^^
+
+With the new characterization tool, the logging code might not be able to keep up with its 5 ms refresh rate thus causing faulty data to be collected.
+
+To see if this is the case, enable print statements on the Driver Station whenever running the data logger. When Autonomous mode is exited, the console will output ``Collected : n in t seconds`` where ``n`` is the number of samples and ``t`` is the time elapsed. If the sampling was successful, ``n`` should equal ``200t`` (rounded).
```
Comment (calcmogul) [inline source/docs/software/wpilib-tools/robot-characterization/viewing-diagnostics.rst:108]: What's `200t`?

**Pull request merged**